app/workflows/same_appeal_substitution_tasks_factory.rb

```diff
@@ -64,25 +118,31 @@ def create_selected_tasks
     end.flatten
   end
 
+  def copy_task(task)
+    new_task_attributes = task.attributes
+      .except(*ATTRIBUTES_EXCLUDED_FROM_TASK_COPY)
+    Task.create!(new_task_attributes)
+  end
+
   def create_task_from(source_task, creation_params)
     case source_task.type
     when "EvidenceSubmissionWindowTask"
-      InitialTasksFactory.new(@appeal).evidence_submission_window_task(source_task, creation_params)
+      evidence_submission_window_task(@appeal, source_task, creation_params)
     when "ScheduleHearingTask"
       distribution_task = @appeal.tasks.open.find_by(type: :DistributionTask)
       ScheduleHearingTask.create!(appeal: @appeal, parent: distribution_task)
     else
-      excluded_attrs = %w[status closed_at placed_on_hold_at]
-      source_task.copy_with_ancestors_to_stream(@appeal, extra_excluded_attributes: excluded_attrs)
+      copy_task(source_task)
     end
   end
 
   def reopen_decision_tasks
-    excluded_attrs = %w[status closed_at placed_on_hold_at]
     if @appeal.tasks.of_type(:AttorneyTask)&.open&.empty? &&
        @appeal.tasks.of_type(:JudgeDecisionReviewTask)&.open&.empty?
```

Comment:
I don't think the safety operators are needed:
```suggestion
    if @appeal.tasks.of_type(:AttorneyTask).open.empty? &&
       @appeal.tasks.of_type(:JudgeDecisionReviewTask).open.empty?
```